app-office/scribus/scribus-1.5.2-r2.ebuild

```diff
@@ -49,14 +49,14 @@ COMMON_DEPEND="
 	media-libs/lcms:2
 	media-libs/libcdr
 	media-libs/libpagemaker
-	media-libs/libpng:0
+	media-libs/libpng:0=
 	media-libs/libvisio
 	media-libs/tiff:0
 	net-print/cups
 	sys-libs/zlib[minizip]
 	virtual/jpeg:0=
 	>=x11-libs/cairo-1.10.0[X,svg]
-	boost? ( dev-libs/boost )
+	boost? ( dev-libs/boost:= )
```

Comment:
Since you are compiling with C++11, you may want to use boost-1.62 as that is the only version at the moment that supports that ABI. The previous versions are only C++98, thus may have some incompatibilities.